Content.Server/ImmovableRod/ImmovableRodSystem.cs

```diff
@@ -44,31 +45,22 @@ public override void Initialize()
         base.Initialize();
 
         SubscribeLocalEvent<ImmovableRodComponent, StartCollideEvent>(OnCollide);
-        SubscribeLocalEvent<ImmovableRodComponent, ComponentInit>(OnComponentInit);
```

Comment:
it should keep the old behaviour when spawned

Reply:
actually it keeps old behaviour - rod does not move when spawned

Reply:
like, rod spawn code is pretty cursed so I got rid of it